src/views/preview/preview.jsx

```diff
@@ -513,8 +528,17 @@ const mapStateToProps = state => {
         userPresent;
     const authorPresent = projectInfoPresent && state.preview.projectInfo.author &&
         Object.keys(state.preview.projectInfo.author).length > 0;
+    const userOwnsProject = isLoggedIn && authorPresent &&
+        state.session.session.user.id === state.preview.projectInfo.author.id;
 
     return {
+        canAddToStudio: isLoggedIn && userOwnsProject,
+        canCreateNew: false,
+        canRemix: false,
```

Comment:
I'd think this would be `!userOwnsProject`.

Reply:
well yes, eventually! but not for this PR. right?

Reply:
Yes, right!